addons/packages/app-toolkit/0.2.0/bundle/config/developer-namespace.yaml

```diff
@@ -0,0 +1,48 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:yaml", "yaml")
+
+#@ if/end data.values.developer_namespace != "default":
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: #@ data.values.developer_namespace
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: registry-credentials
+  namespace: #@ data.values.developer_namespace
+  annotations:
+    secretgen.carvel.dev/image-pull-secret: ""
+type: kubernetes.io/dockerconfigjson
+data:
+  .dockerconfigjson: e30K
+
+---
+apiVersion: v1 
+kind: ServiceAccount
+metadata:
+  name: default
```

Comment:
It's best not to bind additional permissions to the default service account. Workloads should not be run with the same permissions as a supply chain.

Reply:
**Notes from Zoom Call:**
Unfortunately, we're stuck using the default service account until cartographer-catalog fixes a bug where providing a service account to the package actually uses the new service account instead of default. Once that is fixed, we should be able to specify and use a different service account. TAP also does it this way, and that's bad.

Add to backlog and follow-up with cartographer-catalog to track their issue